source/test/TestApp/PlayFabQoSTest.h

```diff
@@ -11,18 +11,10 @@ namespace PlayFabUnit
 {
     struct TestContext;
 
-    // A wrapper around TestEvents, so we can track it back to the test that launched the event
-    class TestEvent : public PlayFab::PlayFabEvent
```

Comment:
Don't remove this until you determine it's not necessary. A properly implemented async pattern on the QoS tests may require it.

Reply:
locally, tests passed without this. 

Jenkins passed https://pf-jenkins.playfab.com/view/SDK/job/SDK_Pipeline_Filtered/3253/console without it.